packages/rainbow-sprinkles/src/defineProperties.ts

```diff
@@ -133,13 +133,18 @@ export type OptionsBoth<
   shorthands?: Shorthands;
 };
 
+export interface CommonOptions {
+  '@layer'?: string;
+}
```

Comment:
Let's use this in `createStyles` and `createStaticStyles`. Move to the `types.ts` file if needed.